hack/install-binaries.sh

```diff
@@ -71,7 +71,7 @@ warn_architecture() {
 
 install_kubectl() {
     echo '=== kubectl'
-    curl -sSLo "${bin}"/kubectl "https://storage.googleapis.com/kubernetes-release/release/v$kubectl_version/bin/linux/${ARCH}/kubectl"
+    curl -sSLo "${bin}"/kubectl "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
```

Comment:
This does not respect kubectl_version and ARCH envvars.

Reply:
im just testing if this works at all just in case